src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int BASE_SIZE = 10;
```

Comment:
It would be better use BASE_CAPACITY name or rename size value to currentAmountofValues